en/clubs/Valberedningen/sidebar.md

```diff
@@ -0,0 +1,5 @@
+## Contact
+
+#### Nominating Committee Chairperson
+Hannah Burak</br>
+[valberedning@d.kth.se](mailto:valberedning@d.kth.se)
```

Comment:
This email should be [valberedning-ordf@d.kth.se](mailto:valberedning-ordf@d.kth.se)

[valberedning@d.kth.se](mailto:valberedning@d.kth.se) should probably still be included, perhaps just below ## Contact?

Same comments apply to Swedish page